test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Wide.Tests.ps1

```diff
@@ -1,35 +1,40 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 Describe "Format-Wide" -Tags "CI" {
+    BeforeAll {
+        1..10 | ForEach-Object { New-Item -Path ("TestDrive:\Testdir{0:00}" -f $_) -ItemType Directory }
+        1..10 | ForEach-Object { New-Item -Path ("TestDrive:\TestFile{0:00}.txt" -f $_) -ItemType File }
+    }
 
     It "Should have the same output between the alias and the unaliased function" {
-        $nonaliased = Get-ChildItem | Format-Wide
-        $aliased    = Get-ChildItem | fw
+        $nonaliased = Get-ChildItem $TestDrive | Format-Wide
+        $aliased = Get-ChildItem $TestDrive | fw
 
         $($nonaliased | Out-String).CompareTo($($aliased | Out-String)) | Should -Be 0
     }
 
     It "Should be able to specify the columns in output using the column switch" {
-        { Get-ChildItem | Format-Wide -Column 3 } | Should -Not -Throw
+        { Get-ChildItem $TestDrive | Format-Wide -Column 3 } | Should -Not -Throw
     }
 
     It "Should be able to use the autosize switch" {
-        { Get-ChildItem | Format-Wide -Autosize } | Should -Not -Throw
-        # Test for #6471
-        { Get-ChildItem | Format-Wide -Autosize | Out-String } | Should -Not -Throw
+        { Get-ChildItem $TestDrive | Format-Wide -Autosize } | Should -Not -Throw
+        # 'Format-Wide -AutoSize | Out-String' fails on PowerShell Core 6.0.1. (issue #6471)
+        # so we add a new test for that.
```

Comment:
Please remove unneeded comment. Each test is important.

Reply:
I will remove it.